src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
```

Comment:
also, you can create a constructor with parameters for the Ball model and simplify your solution.